.github/workflows/asana-release.yml

```diff
@@ -188,7 +188,7 @@ jobs:
         with:
           repository: duckduckgo/duckduckgo-privacy-extension
           path: extensions/
-          ref: develop
+          ref: main
```

Comment:
NAB: Do we always want to be explicit about the branch we're using, or just use the default branch?
If we omit this, it'll use the default branch -- https://github.com/actions/checkout#usage

Reply:
Let's keep it explicit for now.